Week2/homework/js-exercises/ex-3-recipeCard.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+console.log('EXERCISE 3 - The Recipe Card');
+
+/* EXERCISE-3 => 
+First of all we made an object with 3 properties.
+Then we logged out the properties using loop. */
+
+let myMealRecipe = {
+  nameOfMeal: 'meat saute',
+  serves: 4,
+  ingredients: ['1 kg meat', '7 tomatoes', '11 pepper', 'salt'],
+};
+
+console.log(`Meal name: ${myMealRecipe.nameOfMeal}`); // 'Meal name: ' + myMealRecipe.nameOfMeal
+console.log(`Serves: ${myMealRecipe.serves}`);
+console.log('Ingredients:');
+for (let i = 0; i < myMealRecipe.ingredients.length; ++i) {
+  console.log(myMealRecipe.ingredients[i]);
+}
+
+/* Or we can do this with for..of..loop
+
+for (let ingredient of myMealRecipe.ingredients) {
+  console.log(ingredient);
+}
+*/
```

Comment:
👍 but couple things here:
1. You can also use `const` keyword instead of `let` for creating the object since you aren't reassigning it, right? 🙂 
2. Try to lookup for a method on objects called `Object.keys()` and see if it helps in making this implementation easier.

Reply:
Thanks, Yash. I changed it to const. And I also looked at for Object.keys which is a good method as you've mentioned.